circuit/js/src/encoder/axiomV2CircuitMetadata.ts

```diff
@@ -0,0 +1,85 @@
+import { concat, encodePacked, pad } from "viem";
+import { resizeArray } from "../utils";
+import { AxiomV2CircuitMetadataParams } from "../types";
+
+
+export const encodeAxiomV2CircuitMetadata = (params: AxiomV2CircuitMetadataParams): string => {
```

Comment:
i figured i'd build it so it has parity with the rust implementation, but maybe we can enforce the js settings by setting `params.numChallenges = [0]` and `params.isAggregation = false` at the top of the file here, or otherwise have numChallenges and isAggregation be optional parameters w/ the above defaults set if they aren't passed in